.travis/csvtomd.py

```diff
@@ -0,0 +1,66 @@
+"""Convert CSV to HTML file."""
```

Comment:
If the docstring is correct, then the filename should be `csvtohtml.py` or even better `csv2html.py`

Reply:
Missing `usage:` example.